packages/front/svelte/views/association/components/Bodacc/Announcement.controller.js

```diff
@@ -0,0 +1,29 @@
+const MAX_ANNOUCEMENT_NUMBER_SIZE = 5;
+
+export default class AnnouncementController {
+    constructor(announcement) {
+        this.announcement = announcement;
+        this.bodaccType = this.announcement.id.substring(0, 1);
+        this.bodaccNumber = this.announcement.id.substring(1, 9);
+        this.announcementNumber = this.announcement.id.slice(9);
+        this.judgmentObj = JSON.parse(this.announcement.jugement);
+    }
+
+    get judgment() {
+        return this.judgmentObj.complementJugement;
+    }
+
+    get url() {
+        return `https://www.bodacc.fr/pages/annonces-commerciales-detail/?q.id=id:${this.announcement.id}`;
+    }
+
+    get publication() {
```

Comment:
```suggestion
    get publicationFile() {
```

sinon on s'attend pas à ça comme résultat